.travis.yml

```diff
@@ -3,7 +3,7 @@ language: go
 dist: trusty
 
 go:
-  - "1.12"
+  - "1.12.5"
```

Comment:
Would you mind changing this to use whatever the latest point release on the `1.12.x` series is? [Travis docs](https://docs.travis-ci.com/user/languages/go/) say to use `"1.12.x"` for that.

That was my intention in https://github.com/zmap/zlint/commit/64ec0afbd7174756179985c660d7e47141528626 but I goofed and wrote `"1.12"` instead of `"1.12.x"`. 

It looks like CI [has been running](https://travis-ci.org/zmap/zlint/builds/535448296) with exactly `1.12.0` which doesn't make much sense:
> $ travis_setup_go
> go version go1.12 linux/amd64

Reply:
That makes more sense indeed.